scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -73,21 +73,22 @@
           {% else %}
             <tr>
           {% endif %}
-            <td><input type="checkbox" value="{{variant._id}}" name="dismiss" onclick="enableDismiss()"></td>
-            <td class="text-left">{{cell_rank(variant, institute, case, form, manual_rank_options)}}</td>
+            <td>
+              <input type="checkbox" value="{{variant._id}}" name="dismiss" onclick="enableDismiss()">
+            </td>
+            <td>{{ gene_cell(variant) }}</td>
             <td>
               <a href="{{ url_for('variant.variant', institute_id=institute._id, case_name=case.display_name,
                                   variant_id=variant._id, cancer='yes') }}">
-              {% for annotation in variant.canonical_transcripts %}
-                <div>{{ annotation }}</div>
-              {% endfor %}
+                <div>{{ variant.primary_gene.hgvs_identifier }}</div>
+                <div>{{ (variant.primary_gene.hgvsp_identifier or '') |url_decode }}</div>
```

Comment:
Hm, perhaps we should fix this "None" (large indel from demo case) when it is the primary link to the variant:
![Screenshot 2021-04-19 at 18 05 35](https://user-images.githubusercontent.com/758570/115268082-296b3900-a13a-11eb-999a-64223878fbee.png)